util/chunk/chunk.go

```diff
@@ -187,6 +187,11 @@ func (c *Chunk) SwapColumns(other *Chunk) {
 	c.numVirtualRows, other.numVirtualRows = other.numVirtualRows, c.numVirtualRows
 }
 
+// CopyColumns copies columns `other.columns[from]` to `c.columns[dst]`.
+func (c *Chunk) CopyColumns(other *Chunk, dst, from int) {
```

Comment:
How about `MakeRefTo(srcColIdx int, dst *Chunk, dstColIdx int)` and put it together with the implementation of `MakeRef(srcColIdx, dstColIdx int)`?

Reply:
please add a UT for this function.